### ISSUE TITLE
Using ClosedXML, mixed font style in cell is lost

### ISSUE THREAD

**Issue by tarockx** [ClosedXML/ClosedXML#1330]
**Do you want to request a *feature* or report a *bug*?**
- [x] Bug
- [ ] Feature

**Version of ClosedXML**
Tested both 0.94.2 and 0.95.0-beta2

**What is the current behavior?**

I have an XLSX file (attached as [test.xlsx](https://github.com/ClosedXML/ClosedXML/files/4019408/test.xlsx)) that has a cell with mixed font inside it (cell A1). As you can see, the cell has some words in different font styles/colors:
![image](https://user-images.githubusercontent.com/10919460/71728395-4264b380-2e3d-11ea-8a4f-17fd758d9b1a.png)

If I open and save this file using ClosedXML (even without doing anything else), the mixed formatting is lost:
![image](https://user-images.githubusercontent.com/10919460/71728446-5e685500-2e3d-11ea-8714-ca010c9014a2.png)

As you can see, weirdly, the entire cell seems to acquire the style of the first piece of text (bold font in this case)

**What is the expected behavior or new feature?**

Cell styles should not be modified (especially since I didn't even do any modifications to the document, just opened and closed it)

## Reproducibility
I just opened and then saved the document with ClosedXML, like this:

```c#
static void Main(string[] args)
        {
            using (var ms = new MemoryStream(File.ReadAllBytes("test.xlsx")))
            {
                using (var workbook = new XLWorkbook(ms))
                {
                    workbook.SaveAs("test_out.xlsx");
                }

            }
        }
```
- [x] I attached a [sample spreadsheet](https://github.com/ClosedXML/ClosedXML/files/4019408/test.xlsx)

**Comment by igitur** [comment]
Hi,

Your input file contains an interesting case: inline mixed-style content. How was this generated in the first place? I've never seen it. Unfortunately ClosedXML doesn't support this yet, but we can add it. At the moment normal and mixed-styles are supported for shared strings and only normal (non-mixed) inline strings are supported.

We'll add it to our to-do list.

Thanks for filling out the issue template well.

**Comment by tarockx** [comment]
Hi, and thanks for your reply.
The file was generated by Microsoft's Reporting Services engine, its basically what you get when you generate a report in Excel format from either Microsoft sequel server reporting services (SSRS) or the Microsoft's Report Viewer C# library.

**Comment by igitur** [comment]
So I'm guessing that you don't really care whether the strings are stored in the shared string part or whether they're inline, right? If you don't care, the easy solution is that ClosedXML just always saves rich text content as shared strings. @Pankraty ?

**Comment by tarockx** [comment]
Yeah, honestly I don't really care how the document is formatted internally as long as it works 😅️ and thanks again for your quick responses and support, you guys are awesome

**Comment by Pankraty** [comment]
@igitur I don't see reasons against.

**Comment by InvincibleDRT** [comment]
@Pankraty @igitur @tarockx Is this issue resolved? I am having a similar issue. I am rather trying to create an excel file with mixed formatting in a cell.

**Comment by jahav** [comment]
This was fixed long time ago in 0.95.0 and it is still working in `develop` branch. Closing.
